src/db/cards.db.js

```diff
@@ -0,0 +1,75 @@
+let status = ['inqueue', 'inprogress', 'done'];
+let priority = ['Low', 'Medium', 'High'];
+let cardsFromFakeDB = [{
+    id: 1,
```

Comment:
You could leave the id's out since inserting data into that table will autoincrement the PK